Wikipedia/assets/index.js

```diff
@@ -124,7 +124,7 @@ function maybeSendMessageForTarget(event, hrefTarget){
     var href = hrefTarget.getAttribute( "href" );
     var hrefClass = hrefTarget.getAttribute('class');
     if (hrefTarget.getAttribute( "data-action" ) === "edit_section") {
-        window.webkit.messageHandlers.clicks.postMessage({"editClicked": { sectionId: hrefTarget.getAttribute( "data-id" ) }});
```

Comment:
This PR makes the JS not send dictionaries within dictionaries. 
Crashes were happening when these nested dictionaries were NSNull and we tried to access values on them with dictionary subscript syntax - i.e. `dict[@"somekey"]`